src/intervals/trigonometric_functions.jl

```diff
@@ -1,7 +1,9 @@
 # This file is part of the ValidatedNumerics.jl package; MIT licensed
 
-half_pi{T}(::Type{T}) = get_pi(T) / 2
-two_pi{T}(::Type{T})  = get_pi(T) * 2
+half_pi(::Type{Float64})  = @floatinterval(pi/2)
```

Comment:
I agree, it is *safer* (both contain pi/2), but for `Float64`, it is not *as tight as possible*:
```julia
julia> diam(@interval(pi)/2)
6.661338147750939e-16

julia> diam(@interval(pi/2))
2.220446049250313e-16
```
For bigfloat intervals, `@biginterval(pi)/2 == @biginterval(pi/2)` yields true, and corresponds to `@floatinterval(pi/2)`.